TextInputKit/TextInputKit/Code/SwiftExtensions/CountableClosedRange+TextInputKit.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CountableClosedRange+Tests.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 02/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Foundation
+
+extension CountableClosedRange {
```

Comment:
Actually methods in this file and in other files are very valuable.
I ask you to think about moving it to separate public framework.